plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx

```diff
@@ -17,33 +17,41 @@
 import React from 'react';
 import { Entity } from '@backstage/catalog-model';
 import { InfoCard } from '@backstage/core-components';
+import { Grid } from '@material-ui/core';
 import { useDatadogAppData } from './useDatadogAppData';
 import { Resizable } from 're-resizable';
 import ZoomOutMapIcon from '@material-ui/icons/ZoomOutMap';
 
 export const DatadogDashboardPage = ({ entity }: { entity: Entity }) => {
   const { dashboardUrl } = useDatadogAppData({ entity });
+  const allDashboardUrls: string[] = dashboardUrl.split(',');
   return (
-    <InfoCard title="Datadog dashboard">
-      <Resizable
-        defaultSize={{
-          width: '100%',
-          height: 500,
-        }}
-        handleComponent={{ bottomRight: <ZoomOutMapIcon /> }}
-      >
-        <iframe
-          title="dashboard"
-          src={`${dashboardUrl}`}
-          style={{
-            border: 'none',
-            height: '100%',
-            width: '100%',
-            resize: 'both',
-            overflow: 'auto',
-          }}
-        />
-      </Resizable>
-    </InfoCard>
+    <Grid container spacing={3}>
+      {allDashboardUrls.map((value, index) => (
+        <Grid item md={12}>
+          <InfoCard title={`Datadog dashboard ${index}`} variant="gridItem">
+            <Resizable
+              defaultSize={{
+                width: 100,
```

Comment:
should it be reverted to the '100%'?
I have an issue with the default size in the [2.1.15](https://www.npmjs.com/package/@roadiehq/backstage-plugin-datadog/v/2.1.15)

Reply:
More than happy to make that change @stasOR41k  - I think it must have been something I had changed without noticing while playing around with the look and feel